<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NMediaManager::NPreferences::NUi::CKnownAbbreviations</class>
 <widget class="QWidget" name="NMediaManager::NPreferences::NUi::CKnownAbbreviations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Known Abbreviations</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="1">
       <widget class="QToolButton" name="btnAddAbbreviation">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../SABUtils/resources/SABUtils.qrc">
          <normaloff>:/SABUtilsResources/add.png</normaloff>:/SABUtilsResources/add.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="btnDelAbbreviation">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../SABUtils/resources/SABUtils.qrc">
          <normaloff>:/SABUtilsResources/delete.png</normaloff>:/SABUtilsResources/delete.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer_8">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>174</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" rowspan="3">
       <widget class="QTreeView" name="knownAbbreviations"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../SABUtils/resources/SABUtils.qrc"/>
 </resources>
 <connections/>
</ui>
